axelrod/fingerprint.py

```diff
@@ -14,219 +16,355 @@
 Point = namedtuple('Point', 'x y')
 
 
-def create_points(step, progress_bar=True):
-    """Creates a set of Points over the unit square.
+PlayerOrStrategy = Union[Player, type]
+Edges = Tuple[int, int]
+Matches = List[List[Tuple[Action, Action]]]
+
+PointsPlayers = Dict[Point, Player]
+PointsFloats = Dict[Point, float]
+PointsEdges = Dict[Point, Edges]
+PointsMatches = Dict[Point, Matches]
+
+
+class AshlockFingerprint(object):
+    def __init__(self, strategy: PlayerOrStrategy,
+                 probe: PlayerOrStrategy) -> None:
+        self._strategy = strategy
+        self._probe = probe
+        self._data = None  # type: DataOrganizer
+
+    @property
```

Comment:
I changed the af.interactions here to return {Point: matches} instead of {Edge: matches}  because the edges have no meaning beyond the tournament which, by this point has already been discarded.  it seemed more useful to have the tournament results map to the appropriate Points.